src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -590,7 +596,8 @@ private void ParseNamespaceBody(ref SyntaxToken openBrace, ref NamespaceBodyBuil
                                 // incomplete members must be processed before we add any nodes to the body:
                                 AddIncompleteMembers(ref pendingIncompleteMembers, ref body);
 
-                                body.Members.Add(_syntaxFactory.GlobalStatement(ParseUsingStatement()));
+                                body.Members.Add(_syntaxFactory.GlobalStatement(
+                                    attributeLists: default, modifiers: default, ParseUsingStatement()));
```

Comment:
this is the oddest part of teh change currently.  'GlobalStatement' is a 'bridge' node to allow statements to be placed in a compilation unit.  Because GlobalStatement is a member decl, the syntax model now allows for attributes/modifiers.  But we never actually pass these along.  however, we do have the capability to do this in the future if we wanted.

Reply:
Is there a place we could assert that `attributeLists` and `modifiers` are `default` in `GlobalStatement`?

---
In reply to: [252547152](https://github.com/dotnet/roslyn/pull/32999#discussion_r252547152) [](ancestors = 252547152)

Reply:
Added!